fileparse/UnitTest1.cs

```diff
@@ -0,0 +1,42 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for interns at Metamation, Batch - July 2023.
+// Copyright (c) Metamation India.
+// ----------------------------------------------------------------------------
+// Program.cs
+// A10.1:
+// Test cases for filename parser.
+// ------------------------------------------------------------------------------------------------
+namespace Training {
+   [TestClass]
+   public class UnitTest1 {
+      [TestMethod]
+      public void TestMethod1 () {
+         var sTests = new Dictionary<string, bool> {

```

Comment:
Rename the variable to tests because 's' should be used only for static variables.